src/community/index.md.njk

```diff
@@ -45,7 +45,7 @@ When our users make a contribution to the Design System, we like to publicly tha
 
 ### 3. Prioritise openness and honesty
```

Comment:
Not for now, but might need to find another word for 'priority' now that we've given it a specific meaning in 'Upcoming components and patterns' now.